### PR TITLE
bpo-30668: add missing word in license.rst

### PR DESCRIPTION
**Pull request by treebee**
https://bugs.python.org/issue30668

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Review by tirkarthi — APPROVED** [review]
LGTM as noted in the issue. Thank you for your contribution :)

File: Doc/license.rst [hunk 1]
```diff
@@ -561,7 +561,7 @@ SipHash24
 ---------
 
 The file :file:`Python/pyhash.c` contains Marek Majkowski' implementation of
-Dan Bernstein's SipHash24 algorithm. The contains the following note::
+Dan Bernstein's SipHash24 algorithm. The module contains the following note::
```
Comment (matrixise) [inline Doc/license.rst]: Could I suggest to use this sentence. 

```It contains ...```

**Review by matrixise — CHANGES_REQUESTED** [review]

**Review by matrixise — APPROVED** [review]

**Comment by matrixise** [comment]
Congratulation for your merge. Have a nice day and thank you a lot for your contribution.

**Pull request merged**